Src/FluentAssertions/Events/ThreadSafeSequenceGenerator.cs

```diff
@@ -0,0 +1,20 @@
+using System.Threading;
+
+namespace FluentAssertions.Events
+{
+    /// <summary>
+    /// Generates a thread-safe sequence.
```

Comment:
🙃 I'm nitpicking here, but it doesn't generate a thread-safe sequence. It generates a sequence in a thread-safe way...